data/changelog.txt

```diff
@@ -774,6 +798,14 @@ Fix extension cords falling off when items are turned on
 Properly warn if food will cause you to overeat
 Not to shatter already shattered armor.
 Make farm plots re-tillable in farming extension version 2
+Break fake vehicle parts when the real part breaks
+Removes FS2K, Colt Lightning
+updates ench_quadruped_movement_full and the map::player_in_field function to consider quadrupedal movement.
+Tone down hair growth times.
+Reloaded ammo failure removed
+Removes AF-2011A1
+Fixes OM Terrains triggering their exit_eoc on entry instead of correctly triggering their entry_eoc
+Raw copper wire (weighing 4.747 g) can no longer be disassembled into copper (5 g). New recipe to turn raw copper wire into copper.
```

Comment:
```suggestion
Break fake vehicle parts when the real part breaks
Fixes OM Terrains triggering their exit_eoc on entry instead of correctly triggering their entry_eoc
Raw copper wire (weighing 4.747 g) can no longer be disassembled into copper (5 g). New recipe to turn raw copper wire into copper.
```